### PR TITLE
Jhalackova cockpit security

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

Adding 2FA to Cockpit.

### Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### Is this (based on) existing content?

* old xml:id...
* old URL...

### CODE REVIEW

**Comment by lvicoun** [comment]
[Uploading Micro-6.0-cockpit_en.pdf…]()

**Comment by lvicoun** [comment]
[Micro-6.0-cockpit_en.pdf](https://github.com/user-attachments/files/17394775/Micro-6.0-cockpit_en.pdf)

**Review by dariavladykina — COMMENTED** [review]
Hi, please find my suggestions for this one. Thanks!

File: tasks/cockpit-adding-2fa.xml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        <listitem>
+          <itemizedlist>
+            <listitem>
+              <para>
+<link xlink:href="https://psono.com/">PSONO</link> - available for Firefox, Chrome, Docker, iOS, Android
+              </para>
+            </listitem>
+            <listitem>
+              <para>
+                <link xlink:href="https://bitwarden.com/">Bitwarden</link> - available as a Web
```
Comment (JanZerebecki) [inline tasks/cockpit-adding-2fa.xml:51]: Please remove this, as the new version of Bitwarden can not be distributed under a free license anymore https://github.com/gentoo/gentoo/pull/39027 .

**Pull request merged**